cmd/add/smtpConfiguration/smtpConfiguration.go

```diff
@@ -0,0 +1,199 @@
+package smtpConfiguration
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"reflect"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/loginradius/lr-cli/api"
+	"strconv"
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/loginradius/lr-cli/prompt"
+	"github.com/loginradius/lr-cli/config"
+
+
+	"github.com/spf13/cobra"
+)
+
+
+var conf = config.GetInstance()
+
+
+func NewsmtpConfigurationCmd() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "smtp-configuration",
+		Short: "Add a SMTP configuration",
+		Long:  `Configure your SMTP email settings to allow LoginRadius to send email from your email server automatically.`,
+		Example: heredoc.Doc(`
+		$ lr add smtp-configuration
+		SMTP Providers: Mailazy
+		If you don't have a mailazy account. Please Create a mailazy account via https://app.mailazy.com/signup
+		? Key: <Key>
+		? Secret: <Secret>
+		? From Name: <Name>
+		? From Email Id: <Email ID>
+		
+		SMTP settings are saved
+		
+		? Send an email to verify your configuration settings are correct (Y/N): Yes
+		? To Email: <Email ID for Verification>
+		SMTP settings are verified
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return addAccessRestriction()
+		},
+	}
+
+	return cmd
+}
+
+func addAccessRestriction() error {
+	var smtpSchema api.SmtpConfigSchema 
+	var smtpLabels = [] string {"Provider","Key","Secret", "SmtpHost", "SmtpPort",
+	"FromName","FromEmailId","UserName","Password",  "IsSsl"}
+	var smtpProviders []string 
+	var num int
+	var isSsl bool
+	var isMailazy bool 
+	num = len(cmdutil.SmtpProviders)
+	for i := 0; i < num; i++ { 
+		smtpProviders = append(smtpProviders, cmdutil.SmtpProviders[i].Name)
+	}
+	for _,val := range smtpLabels {
+		
+		if val == "Provider" {
+			err := prompt.SurveyAskOne(&survey.Select{
+				Message: cmdutil.SmtpOptionNames[val] + " :",
+				Options:  smtpProviders,
+				}, &num)
+				if err != nil {
+					return err
+				}					
+		} else if val == "IsSsl" && num != 0 {
+			isSsl = cmdutil.SmtpProviders[num].EnableSSL
+			if err := prompt.Confirm(cmdutil.SmtpOptionNames[val] + ":", 
+						&isSsl); err != nil {
+							return err
+			}
+		} else {
+			if num == 0 {
+				isMailazy = ( val != "Password" && val != "IsSsl" && val != "UserName" && val != "SmtpHost" && val != "SmtpPort")
+			} else if num == 9 {
+				isMailazy = (val != "Key" && val != "Secret" )
+			} else if num != 0 {
+				isMailazy = (val != "Key" && val != "Secret" && val != "SmtpHost" && val != "SmtpPort")
+			}
```

Comment:
We can create this as a util funciton

Reply:
Noted..Will add this in next improvement.